starboard/build/config/BUILDCONFIG.gn

```diff
@@ -377,6 +377,90 @@ template("shared_library") {
       }
     }
 
+    if (sb_is_evergreen_compatible && sb_evergreen_compatible_package && target_name != "loader_app") {
+      original_target_name = target_name
+      data_deps += [ "//third_party/icu:icudata" ]
+      data_deps += [
+        ":copy_loader_app_lib${original_target_name}",
+        "//starboard/loader_app:copy_loader_app_content",
+      ]
+      if (cobalt_font_package == "empty") {
+        data_deps += [ "//cobalt/content/fonts:copy_font_data" ]
+      } else {
+        data_deps += [
+          "//cobalt/content/fonts:copy_fonts",
+          "//cobalt/content/fonts:fonts_xml",
+        ]
+      }
+      shared_library(original_target_name + "_evergreen_loader") {
```

Comment:
The loader is a shared library, not an executable? Is this because you're specifically looking at android?

Reply:
I think we should make this more general, as we'll eventually want linux to do something similar. If it's a lot of work we can wait on it, but if it's easy let's do it now

Reply:
lgtm